config/github_repos.json

```diff
@@ -31,6 +31,7 @@
   "emurgo/yoroi",
   "emurgo/yoroi-frontend",
   "emurgo/yoroi-mobile",
+  "github.com/fivebinaries/go-cardano-serialization",
```

Comment:
@wolf31o2 Thanks for the contribution! Could you remove the domain part though and then move the line to be alphabetically ordered again? ❤️

Reply:
D'oh. Yeah, I'll fix it, shortly. I got too excited.

Reply:
Awesome, thanks :)

Reply:
Fixed